src/sparsezoo/v2/helpers.py

```diff
@@ -0,0 +1,200 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Helper functions for SparseZoo v2
+"""
+
+import copy
+from typing import Any, Dict, List
+
+
+def restructure_request_json(
+    request_json: List[Dict[str, Any]],
+    clone_sample_outputs: bool = True,
+    tars: bool = True,
+) -> Dict[str, Any]:
+    """
+    Takes the raw API response and restructures it, so that the output mocks the
+    planned, future structure of the ModelDirectory.
+
+    :params request_json: the output from NeuralMagic API
+    :params clone_sample_outputs: whether we should convert `sample_outputs` folder
+        to `sample_outputs_deepsparse` and `sample_outputs_onnxruntime`
+    :params tars: whether to represent directories within ModelDirectory as
+        archived folders or loose files
+    :return: restructured request_json
+    """
+
+    if tars:
+        request_json = _create_tar_directories(request_json)
+
+    else:
+        # create `training` and `deployment` files
+        data = _fetch_from_request_json(request_json, "file_type", "framework")
+        for (idx, file_dict) in data:
+            training_file_dict, deployment_file_dict = file_dict, copy.copy(file_dict)
+            training_file_dict["file_type"], deployment_file_dict["file_type"] = (
+                "training",
+                "deployment",
+            )
+            request_json[idx] = training_file_dict
+            request_json.append(deployment_file_dict)
+
+        # append `model.onnx` to `deployment` files
+        data = _fetch_from_request_json(request_json, "display_name", "model.onnx")
+        assert len(data) == 1
+        _, file_dict = data[0]
+        file_dict["file_type"] = "deployment"
+        request_json.append(file_dict)
+
+        # create files for onnx directory (use `model.onnx` to simulate them)
+        files_to_create = [
+            "model.11.onnx",
+            "model.14.onnx",
+        ]
```

Comment:
what is the goal here?

Reply:
why are we creating dummy files in `src` instead of `tests`?

Reply:
this has been resolved.